optimum/exporters/onnx/base.py

```diff
@@ -213,53 +177,12 @@ def __init__(
         float_dtype: str = "fp32",
         legacy: bool = False,
     ):
-        self.task = task
-        self.int_dtype = int_dtype
-        self.float_dtype = float_dtype
+        super().__init__(config=config, task=task, int_dtype=int_dtype, float_dtype=float_dtype)
 
-        self._config = config
-        self._preprocessors = preprocessors
-        self._normalized_config = self.NORMALIZED_CONFIG_CLASS(self._config)
         self.variant = "default"
+        self._preprocessors = preprocessors
```

Comment:
could be moved to the base config as well (the way i see it is that information about the model is sometimes scattered across config and preprocessors)